README.md

```diff
@@ -16,7 +16,7 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * **xVhd** manages VHDs in a Hyper-V host. 
 * **xVMHyperV** manages VMs in a Hyper-V host.
 * **xVMSwitch** manages virtual switches in a Hyper-V host. 
-* **xVhdFileDirectory** manages files or directories in a VHD. 
+* **xVhdFile** manages files or directories in a VHD. 
```

Comment:
I would say we update it to write both class and friendly name in readme file. It will make it easier to navigate to the corresponding resource in DSCResources directory. something like xVhdFileDirectory(xvhdFile).